build/npm/postinstall.js

```diff
@@ -23,9 +23,17 @@ function yarnInstall(location, opts) {
 	const args = original.filter(arg => arg === '--ignore-optional' || arg === '--frozen-lockfile');
 
 	console.log(`Installing dependencies in ${location}...`);
+	const oldConfigArch = process.env['npm_config_arch']
+	if (location.startsWith("remote") && oldConfigArch == "arm64") {
+		// Temporarily set remote arch to x86, because remote portion builds
+		// against node, which is not consistently supported on Windows on Arm
+		// (yet).
+		process.env["npm_config_arch"] = "ia32"
+	}
 	console.log(`$ yarn ${args.join(' ')}`);
 	const result = cp.spawnSync(yarn, args, opts);
```

Comment:
Shouldn't the new environment flag simply be added to the `opts`, instead of fiddling around with the current process env?

Reply:
Agreed, setting the env through `opts` will be cleaner.